stanford-earth/SE3_BLT
# Changes to Composer/settings/BLT

# READY FOR REVIEW


# Summary
- Revised patching of settings.php to match BLT's standards
- Changed name of custom.settings.php file to includes.settings.php to match BLT standards
- Removed, scaffolded, and adds BLT's settings.php patch on composer update/install
- Now requires your local environment to use the local.settings.php found in `/sites/default/settings/local.settings.php` or changes will get blown away.

# Needed By (Date)
- Sooner the better. Current implementation causes code duplication on settings.php file.

# Steps to Test

1. Check out this branch
2. Trash `/docroot` << Please save all of your work in progress first
3. Run `composer install --prefer-source`
4. Add your DB settings to `/docroot/sites/default/settings/local.settings.php`
5. Run `blt sync:refresh -Dsync.files=true`
6. Validate everything is in place and working (saml link, site loads, .htaccess was patched)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
